index.bs

```diff
@@ -322,8 +322,8 @@ stream|canceling=] the stream, or [=piping=] the readable stream to a writable s
 acquired via the stream's {{ReadableStream/getReader()}} method.
 
 A [=readable byte stream=] has the ability to vend two types of readers: <dfn export lt="default
-reader">default readers</dfn> and <dfn export lt="BYOB reader">BYOB readers</dfn>. BYOB ("bring your
-own buffer") readers allow reading into a developer-supplied buffer, thus minimizing copies. A
+reader">default readers</dfn> and <dfn export lt="BYOB reader" for="ReadableStream">BYOB readers</dfn>.
```

Comment:
Why do we need to scope it to `ReadableStream`?

Reply:
Ah my bad, I thought I needed it, but it turns out I can just reference it without the scope :)